packages/sanity/src/_internal/cli/commands/backup/listBackupCommand.ts

```diff
@@ -0,0 +1,132 @@
+import type {CliCommandDefinition} from '@sanity/cli'
+import {Table} from 'console-table-printer'
+import {lightFormat} from 'date-fns'
+import moment from 'moment'
+import resolveApiClient from '../../actions/backup/resolveApiClient'
+import {defaultApiVersion, validateLimit} from './backupGroup'
+
+const DEFAULT_LIST_BACKUP_LIMIT = 30
+
+interface ListDatasetBackupFlags {
+  before?: string
+  after?: string
+  limit?: string
+}
+
+type ListBackupRequestQueryParams = {
+  start?: string
+  end?: string
```

Comment:
I've made these `after` and `before` in the API to match the CLI.

Reply:
Updated the query param names.